rosdep/python.yaml

```diff
@@ -3200,25 +3201,32 @@ python3-babeltrace:
     wily: [python3-babeltrace]
     xenial: [python3-babeltrace]
 python3-empy:
+  gentoo: [dev-python/empy]
   ubuntu:
     xenial: [python3-empy]
 python3-flake8:
+  gentoo: [dev-python/flake8]
   ubuntu:
     xenial: [python3-flake8]
 python3-pep8:
+  gentoo: [dev-python/pep8]
   ubuntu:
     xenial: [python3-pep8]
 python3-pyparsing:
+  gentoo: [dev-python/pyparsing]
   ubuntu:
     xenial: [python3-pyparsing]
 python3-nose:
+  gentoo: [dev-python/setuptools]
```

Comment:
typo?

Reply:
typo/brain-malfunction, actually (and sadly). Thanks.